payments-core/src/main/java/com/stripe/android/cards/CardAccountRangeService.kt

```diff
@@ -43,17 +43,6 @@ class CardAccountRangeService(
             return
         }
 
-        val testAccountRanges = if (isCbcEligible()) {
```

Comment:
This was used to ensure that the test cards were handled properly in this service. Now the test cards are part of the static card ranges (queried below), so we don't need this special handling.